tests/functional/test_single_model_vehicle.py

```diff
@@ -51,23 +61,150 @@ def test_run_inference(self, vehicle_adas_model_downloader,
 
         """
 
-        _, ports = start_server_single_vehicle__model
-        print("Downloaded model files:", vehicle_adas_data_downloader)
+        _, ports = start_server_single_vehicle_model
 
         # Connect to grpc service
         stub = create_grpc_channel('localhost:{}'.format(ports["grpc_port"]),
                                    PREDICTION_SERVICE)
 
-        imgs_v1_224 = np.ones((1, 3, 224, 224))
-        in_name = 'map/TensorArrayStack/TensorArrayGatherV3'
-        out_name = 'softmax_tensor'
-        output = infer(imgs_v1_224, input_tensor=in_name, grpc_stub=stub,
-                       model_spec_name='resnet',
+        imgs_v1_384 = np.ones((1, 3, 384, 672))
+        in_name = 'data'
+        out_name = 'detection_out'
+        output = infer(imgs_v1_384, input_tensor=in_name, grpc_stub=stub,
+                       model_spec_name='vehicle-detection',
                        model_spec_version=None,
                        output_tensors=[out_name])
         print("output shape", output[out_name].shape)
-        assert output[out_name].shape == (1, 1001), ERROR_SHAPE
+        assert output[out_name].shape == (1, 1, 200, 7), ERROR_SHAPE
+
+
+    def test_run_inference_img(self, vehicle_adas_model_downloader,
+                           vehicle_adas_data_downloader,
+                           start_server_single_vehicle_model,
+                           create_grpc_channel):
+        """
+        <b>Description</b>
+        Submit request to gRPC interface serving a single vehicle model
+
+        <b>input data</b>
+        - directory with the model in IR format
+        - docker image with ie-serving-py service
+
+        <b>fixtures used</b>
+        - model downloader
+        - service launching
+
+        <b>Expected results</b>
+        - response contains proper numpy shape
+
+        """
+
+        _, ports = start_server_single_vehicle_model
+        imgs_path =  os.path.join(vehicle_adas_data_downloader, "data", "annotation_val_images")
+
+        img_files = os.listdir(imgs_path)
+        imgs = np.zeros((0,3,384,672), np.dtype('<f'))
+        input_img = self.load_image(os.path.join(imgs_path,"image_000015.jpg"), 672, 384)
+        imgs = np.append(imgs, input_img, axis=0)
```

Comment:
You could go with:
```
input_img = self.load_image(os.path.join(img_path,"image_000015.jpg"), 672, 384)
img = np.expand_dims(input_img, axis=0)
```
If you just want to have additional dimension.

Reply:
Done.